docs/reference/enrichement.mdx

```diff
@@ -0,0 +1,652 @@
+---
+title: Enrichment
+sidebar_position: 350
+description: >
+  Describe the automatic enrichment Inspektor Gadget performs with the
+  associated Kubernetes and container runtime information.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The enrichment process ensures that every time a Gadget reports an event
+generated within a container, the Gadget data is automatically enriched with the
+associated container information. This information can be divided into two
+categories:
+
+- [Kubernetes-specific information](#kubernetes-enrichment): The Kubernetes
+  information associated with the container where the event was generated, such
+  as the node name, namespace, pod name, etc.
+- [Container Runtime-specific information](#container-runtime-enrichment): The
+  container information, at the runtime level, associated with the container
+  where the event was generated, such as the container ID, image name and
+  digest, etc.
+
+Having this information automatically available is crucial for debugging and
+observability tasks because it allows you to correlate low-level events with
+their higher-level context.
+
+## Kubernetes Enrichment
+
+Inspektor Gadget can enrich Gadget data with the following Kubernetes-specific
+information:
+
+- Node name
+- Namespace
+- Pod name
+- Container name
+- [Owner reference](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/)
+- Pod labels and annotations are work in progress, see issue
+  [#2318](https://github.com/inspektor-gadget/inspektor-gadget/issues/2318)
+
+<Tabs groupId="env">
+<TabItem value="kubectl-gadget" label="kubectl gadget">
```

Comment:
I'm not sure that using "tabs" for this document is a good idea. IMO it makes it more difficult to read as there is so much text on each tab. 

What about having those as different sections:

```
### Kubernetes 

#### kubectl gadget 

#### ig 

### Container Runtime

#### kubectl-gadget 

#### ig 
```